packages/react-components/src/components/InputBoxComponent.tsx

```diff
@@ -130,28 +130,30 @@ export type InputBoxButtonProps = {
   onClick: (e: React.MouseEvent<HTMLDivElement, MouseEvent>) => void;
   className?: string;
   id?: string;
+  ariaLabel?: string;
```

Comment:
Instead, `InputBoxButtonProps` should inherit from `IButtonProps` and pass through those props to the `IconButton`.

Reply:
after discussion and the fact that onRenderIcon is not defined the same way between the 2 props due to rendering different icon depending on hovering state, I will not do this inheritance.
I will indeed revisit the API though but in a following PR